### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by arun-kumar-codes**

### CODE REVIEW

File: src/app/client/actions/users/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+export function loginUser(email,password){
+	return function(dispatch,getState){
+		return new Promise((resolve,reject)=>{
+			Meteor.loginWithPassword(email,password,(error)=>{
+				if(error){
+					reject(error)
+				}else{
+					let user={
+						email:email,
+						password:password
```
Comment (arun-kumar-codes) [inline src/app/client/actions/users/index.js:14]: can we get user id also here

File: src/app/client/components/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    }
+	constructor(props){
+		super(props);
+		this.state={
+			email:'',
+			password:''
+		}
+		this.loginUser=this.loginUser.bind(this);
+	}
+	/*componentWillMount(){
```
Comment (arun-kumar-codes) [inline src/app/client/components/Login.jsx]: reove commented code

File: src/app/client/components/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+		return(
+			<div>
+		     <AppBar 
+		     title="Login" 
+		     iconElementRight={<RaisedButton label="LogIn" 
+		     onTouchTap={this.openLogin}/>}
+		     />
+		     <div style={{'textAlign':'center'}}>
+		     <div style={{'width':'400','paddingTop':'5px','margin':'0px auto'}}>
+		     <h3>Login</h3>
```
Comment (arun-kumar-codes) [inline src/app/client/components/Login.jsx:74]: need to add proper css, from theme

File: src/app/client/components/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+				<TextField value={this.state.password} style={{'width':'80%'}} floatingLabelText="Password"
+				onChange={
+					(e)=>{
+						this.setState({
+							password:e.target.value
+						})
+					}
+				}/>
+				</div>
+				<div style={{'textAlign': 'left','marginLeft':'20px'}}>
```
Comment (arun-kumar-codes) [inline src/app/client/components/Login.jsx]: code formatting missing

File: src/app/client/containers/Login.jsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  render(){
+		return(
+			<Login 
+			router={this.props.router} 
+			ui={this.props.ui} 
+			onLogin={this.props.login}/>
+			);
+	}
+}
+LoginContainer.childContextTypes = {
```
Comment (arun-kumar-codes) [inline src/app/client/containers/Login.jsx:27]: format code

File: src/app/client/index.jsx [hunk 6]
```diff
@@ -4,20 +4,28 @@ import Immutable from 'immutable';
...
 import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import LoginContainer from './containers/Login';
 
 export let isMonitorAction;
 
 const logger = createLogger();
 injectTapEventPlugin();
 
+
+console.log('aaaa')
```
Comment (arun-kumar-codes) [inline src/app/client/index.jsx]: no console logs in pull

File: src/app/client/reducers/entries/users.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import {LOGIN,GET_USER} from '../../actions/users';
+import Immutable from 'immutable';
+export function users(state = Immutable.fromJS({}),action){
```
Comment (arun-kumar-codes) [inline src/app/client/reducers/entries/users.js]: there needs to be a combine reducers for entites as well

File: src/app/collections/index.js [hunk 8]
```diff
@@ -0,0 +1,4 @@
+import { Mongo } from 'meteor/mongo';
```
Comment (mageexxcel) [inline src/app/collections/index.js]: u can delete this file if no need

**Pull request merged**